starter-bots/scala/bot.json

```diff
@@ -0,0 +1,8 @@
+{
+  "author": "Johann Venter",
+  "email": "johannventer2@gmail.com",
+  "nickName": "Johann",
+  "botLocation": "/target/scala-2.12",
+  "botFileName": "entelect-scala-bot_2.12-0.1.jar",
+  "botLanguage": "scala"
+}
```

Comment:
Starter bots will be used by anyone in the community to build their bots, we need to remove personal information from these.
Please change it to:
```
	"author": "John Doe",
	"email": "john.doe@example.com",
	"nickName": "Odersky",
```
we use the designer/creator of the language's surname as the nickname in the starter-bot, in recognition of them. 
P.S, just double check this is correct, Scala was created by "Martin Odersky", right?